common/persistence/src/main/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyBuilder.java

```diff
@@ -144,16 +147,28 @@ public FinalBuilder withDaysSinceOnsetOfSymptoms(int daysSinceOnsetOfSymptoms) {
     return this;
   }
 
+  @Override
+  public FinalBuilder withFieldNormalization(DiagnosisKeyNormalizer fieldNormalizer) {
```

Comment:
some helpful JavaDoc maybe?

Reply:
The javadoc was added to the inherited interface. See FinalBuilder